source/tutorials/integration-testing-using-virtual-machines.md

```diff
@@ -1,42 +1,207 @@
 (integration-testing-vms)=
 
-# Integration testing using virtual machines (VMs)
+# NixOS Configuration testing
```

Comment:
For a beginner coming to Nix ecosystem this doesn't mean anything, while the old wording uses industry established keywords that give an idea what this tutorial is about.

What's the motivation for the title change?